components/patient-access-rules/api/src/main/resources/ApplicationResources.properties

```diff
@@ -33,7 +33,10 @@ phenotips.patientAccessRightsManagement.heading=Access rights for __patientID__
 phenotips.patientAccessRightsManagement.noOwner=nobody
 phenotips.patientAccessRightsManagement.selfOwner=you
 phenotips.patientAccessRightsManagement.caseOwnerandVisibility=This case is owned by {0} and is {1}.
-phenotips.patientAccessRightsManagement.collaboratorsSummary=It is shared with {0} {1,choice,0#no collaborators|1#1 collaborator|1<{1} collaborators} {2}
+phenotips.patientAccessRightsManagement.collaboratorsSummary=It is shared with
+phenotips.patientAccessRightsManagement.collaboratorsSummary.noCollaborators=no collaborators
+phenotips.patientAccessRightsManagement.collaboratorsSummary.oneCollaborator={0} and {1} other collaborator
+phenotips.patientAccessRightsManagement.collaboratorsSummary.xCollaborators={0} and {1} other collaborators
```

Comment:
There's no guarantee that this breakdown into pieces will work in some other languages. It certainly won't work in German, where the word for "shared" needs to go at the very end of the sentence (the order of the words would be "It is with {.....} shared"). So always translate full sentences, not pieces if sentences. If the variation with the parameters is too complex, use different keys and do `if`s in velocity. This is a blocker issue.